app/models/payment/go_cardless/subscription.rb

```diff
@@ -0,0 +1,74 @@
+class Payment::GoCardless::Subscription < ActiveRecord::Base
+  class Charge
```

Comment:
This is rather unconventional, it would be nice to have a comment or two about what's going on here.